src/lib.rs

```diff
@@ -87,6 +87,19 @@ pub struct ProgramBinary {
     pub format: u32,
 }
 
+/// A trait for types that can be used as a context for OpenGL, OpenGL ES, and WebGL functions.
+///
+/// This trait is sealed and cannot be implemented outside of this crate.
+///
+/// # Safety
+///
+/// All GL API usage must be valid. For example, each function call should follow the rules in the
+/// relevant GL specification for the type of context being used. This crate doesn't enforce these
+/// rules, so it is up to the caller to ensure they're followed.
+///
+/// The context implementing this trait must be current when it is dropped. This is necessary to
+/// ensure that certain context state can be deleted on the correct thread. Usually this is only
+/// a concern for desktop GL contexts that are shared between threads.
```

Comment:
> Usually this is only a concern for desktop GL contexts that are shared between threads.

I assume this is because the context is typically always kept current when working with it in a single thread.